### ISSUE TITLE
How to disable rules in pre-commit setup

### ISSUE THREAD

**Issue by jarkenau** [igorshubovych/markdownlint-cli#447]
Is there a way to disable rules globally when using `pre-commit`? 

Copying the syntax from the `npm` package does not work.

```shell
 markdownlint --disable MD013  -- docs/*
```
Pre-commit config:
```yaml
- repo: https://github.com/igorshubovych/markdownlint-cli
    rev: v0.38.0
    hooks:
    - id: markdownlint
      args: [--disable MD013 -- ]
```
Results in:

```shell
markdownlint.............................................................Failed
- hook id: markdownlint
- exit code: 1

error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
error: unknown option '--disable MD013 --'
``

**Comment by DavidAnson** [comment]
According to the pre-commit documentation, it looks like "args" are meant to be separated with a comma: https://pre-commit.com/#arguments-pattern-in-hooks

In that same section, it suggests your scenario could actually use "entry" instead.

Finally, you could use a ".markdownlint.json" configuration file to disable that rule (including for other tools) instead of passing it as an argument.

**Comment by jarkenau** [comment]
Thanks for the hints. To disable multiple rules, I got these two combinations to work:

Via `.pre.commit-config.yaml`:
```yaml
repo: https://github.com/igorshubovych/markdownlint-cli
    rev: v0.38.0
    hooks:
    - id: markdownlint
      args: [--disable=MD033, --disable=MD013, --disable=MD007]
```

Or via `.markdownlint.json`:
```json
{
    "MD007": false,
    "MD033": false,
    "MD013": false
}
```

I did not try the `entry` approach you described, as I don't have a local hook.
